recipe/meta.yaml

```diff
@@ -21,18 +21,18 @@ requirements:
     run:
         - python
         - pandas
+        - six
+        - gdal >=2.*
         - pyproj
         - shapely
         - fiona
-        - gdal 1.11.4
-        - six
         - rtree
-        - descartes
-        - matplotlib
-        - psycopg2  # [not win]
-        - sqlalchemy
-        - geopy ==1.10.0
+        - geopy 1.10.0
         - pysal
+        - matplotlib  # [not win and py35]
```

Comment:
`gdal` is compiled against `jpeg 9*`, but `matplotlib`'s `qt` is pinned against `jpeg 8*`. I believe that is the inconsistency that is here. What puzzles me is why this is affecting only python 3.5 on Windows.